### PR TITLE
Rutherford2024mods

### PR DESCRIPTION
**Pull request by svisak**
Many updates, both new features and compatibility updates as well as updated documentation.

### CODE REVIEW

**Comment by drannawatts** [comment]
I've removed the conflict on the acknowledgements file (hopefully)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -33,6 +33,49 @@ and this project adheres to
...
+^^^^^^^
+* Dependencies: getdist, alive_progress
+
+* Functionality: the option to plot two distributions at the same time using neost.PosteriorAnalysis.mass_radius_prior_predictive_plot().
+
+Changed
+^^^^^^^
+* Cython TOV solvers no longer print "using c code". Python TOV solvers, on the other hand, now raise a warning when they are in use.
+* Renamed build.py to setup.py to enable installing with pip
+
```
Comment (drannawatts) [inline CHANGELOG.rst:71]: Add here 'Updated core team membership'

File: docs/source/install.rst [hunk 2]
```diff
@@ -13,132 +13,71 @@ and is available on `GitHub`_ and can be cloned as:
 
 	git clone https://github.com/xpsi-group/neost.git </path/to/neost>
 
-On this page we will describe how to install NEoST and it's prerequisites 
+On this page we will describe how to install NEoST and it's prerequisites
```
Comment (drannawatts) [inline docs/source/install.rst:16]: its not it's

File: docs/source/install.rst [hunk 3]
```diff
@@ -3,8 +3,8 @@
 Installation
 ============
 
-NEoST is a Python software package that performs neutron star equation 
-of state inference process. it is an open source package that was 
+NEoST is a Python software package that performs neutron star equation
+of state inference process. it is an open source package that was
```
Comment (drannawatts) [inline docs/source/install.rst:7]: Capital I at start of 2nd sentence

File: neost/__init__.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 """ NEoST: Open-source code for equation of state inference via nested sampling. """
 from __future__ import print_function
-__version__ = "0.9.0"
+__version__ = "0.10.0"
 __author__ = "Geert Raaijmakers"
```
Comment (drannawatts) [inline neost/__init__.py:4]: I think you should change author to "The NEoST core team'.  This is consistent with what we did for X-PSI and also far more accurate!!

File: CHANGELOG.rst (hunk 1 above)
Comment (svisak) [reply CHANGELOG.rst:71]: Done

File: docs/source/install.rst (hunk 2 above)
Comment (svisak) [reply docs/source/install.rst:16]: Done

File: docs/source/install.rst (hunk 3 above)
Comment (svisak) [reply docs/source/install.rst:7]: Done, also changed "was and is" to "is". It's not really important that NEoST was available on github in the past :D

**Comment by nr1118** [comment]
This all looks good to me!

**Comment by nr1118** [comment]
Do I need to approve these changes one by one or is there  one click option?

**Comment by svisak** [comment]
> Do I need to approve these changes one by one or is there one click option?

Not sure.

**Review by nr1118 — APPROVED** [review]

**Pull request merged**